### PR TITLE
chore: add IT for dialog shortcut and writing

### PR DESCRIPTION
**Pull request by pleku**
When a shortcut is added with listenOn(dialog), writing to an input
(like TextField) inside the dialog should not be blocked.

Related to vaadin/flow#10362, depends on vaadin/flow#10388

### CODE REVIEW

**Review by sosa-vaadin — APPROVED** [review]

File: vaadin-dialog-flow-parent/vaadin-dialog-flow-integration-tests/src/main/java/com/vaadin/flow/component/dialog/tests/DialogWithShortcutPage.java [hunk 1]
```diff
@@ -7,23 +7,27 @@
...
+import com.vaadin.flow.component.combobox.ComboBox;
+import com.vaadin.flow.component.dependency.NpmPackage;
 import com.vaadin.flow.component.dialog.Dialog;
 import com.vaadin.flow.component.html.Div;
 import com.vaadin.flow.component.html.Input;
 import com.vaadin.flow.component.html.NativeButton;
 import com.vaadin.flow.component.orderedlayout.VerticalLayout;
 import com.vaadin.flow.router.Route;
 
+@NpmPackage(value="@vaadin/vaadin-text-field", version = "3.0.2")
```
Comment (alvarezguille) [inline vaadin-dialog-flow-parent/vaadin-dialog-flow-integration-tests/src/main/java/com/vaadin/flow/component/dialog/tests/DialogWithShortcutPage.java:19]: Why is this needed? If this can't be avoided it should be updated and we should also make sure that the script updating WC versions will pick this one as well.

**Review by alvarezguille — CHANGES_REQUESTED** [review]

**Comment by tulioag** [comment]
Hi @pleku . Is this still relevant? Can we remove the `@NpmPackage(value="@vaadin/vaadin-text-field", version = "3.0.2")` annotation or is it necessary?

File: vaadin-dialog-flow-parent/vaadin-dialog-flow-integration-tests/src/main/java/com/vaadin/flow/component/dialog/tests/DialogWithShortcutPage.java (hunk 1 above)
Comment (pleku) [reply vaadin-dialog-flow-parent/vaadin-dialog-flow-integration-tests/src/main/java/com/vaadin/flow/component/dialog/tests/DialogWithShortcutPage.java]: Done

**Comment by pleku** [comment]
@tulioag I don't know where the annotation came from - it is removed now.

**Review by alvarezguille — APPROVED** [review]
